drivers/LVHDSR.py

```diff
@@ -2232,6 +2231,7 @@ def _activate_cbt_log(self, lv_name):
             except Exception as e:
                 util.SMlog("Exception in _activate_cbt_log, "
                            "Error: %s." % str(e))
+                raise
```

Comment:
`raise from e` would avoid loosing original exception, and makes it unnecessary to format it ourselves

Reply:
raise on it's own re-raises the original exception

Reply:
ah yes, my bad